airflow/www/jest-setup.js

```diff
@@ -20,9 +20,6 @@
  */
 
 import '@testing-library/jest-dom';
-import { enableFetchMocks } from 'jest-fetch-mock';
-
-enableFetchMocks();
```

Comment:
I realized that we could just mock this call ourselves and we didn't need this package.

Reply:
Any `leftpad` around :smile:  ?